### DISCUSSION TITLE
Why did they remove the ability to copy and paste?

### DISCUSSION THREAD

**Discussion by ghost** [codewars/codewars.com#2705] [General]
I solve problems in Visual Studio Code and then paste them into codewars. I'm not comfortable solving problems without a normal consol.log(). Return this feature to my account. horror

**Comment by hobovsky** [comment]
They did not remove the ability to copy and paste.

You most probably have accidentally enabled VIM mode: <https://docs.codewars.com/references/kata-trainer#editors>. Just disable it to be able to use non-VIM key bindings again.

**Reply by ghost** [reply]
ohh thank you man ahahha
